big_tests/tests/mod_event_pusher_rabbit_SUITE.erl

```diff
@@ -582,6 +590,13 @@ stop_mod_event_pusher_rabbit() ->
     Host = ct:get_config({hosts, mim, domain}),
     rpc(mim(), gen_mod, stop_module, [Host, mod_event_pusher_rabbit]).
 
+start_wpool(Host) ->
+    rpc(mim(), mongoose_wpool, ensure_started, []),
+    rpc(mim(), mongoose_wpool, start_configured_pools, [[?WPOOL_CFG], [Host]]).
+
+stop_wpool(Host) ->
```

Comment:
Again, the name is misleading. It stops only the rabbit pool, not the entire `mongoose_wpool`.